src/TabView.tsx

```diff
@@ -36,6 +36,10 @@ interface Props<Route extends BaseRoute> {
    * Whether to disable page animations between tabs. (iOS only)
    */
   disablePageAnimations?: boolean;
+  /**
+   * Describes the appearance attributes for the tabBar to use when an observable scroll view is scrolled to the bottom. (iOS only)
+   */
```

Comment:
Can you also add this to the docs? 

https://github.com/okwasniewski/react-native-bottom-tabs/blob/main/docs/docs/docs/guides/usage-with-react-navigation.md